cypress/e2e/counter.cy.ts

```diff
@@ -0,0 +1,21 @@
+describe("Counter", () => {
+  it("increments and decrements the counter correctly", () => {
```

Comment:
Good work.
Not for refactoring (as later in the course you will provide the e2e tests for the whole page - what is actually the way it should be):
- Good to check if button exist before clicking
- Good to try different initial values
- As usual, e2e includes several tests, => good to use beforeEach hook

Reply:
Added check if exists. Here problem with setting initial values. How to do it?

Reply:
It can be implemented in several ways.
Need to use a loop to execute the test with different values.
Smth like this:
const initialValues = [0, 1, 2, 3, 4, 5];

initialValues.forEach((initialValue) => {
  describe(`Counter with initial value ${initialValue}`, () => {
    beforeEach(()=>{
      cy.visit("http://localhost:3000/");
      
      for(let i = 0; i < initialValue; i++) {
        cy.get('button').contains('+').click();
      }
    });

    it("increments and decrements the counter correctly", () => {
      cy.get("div").contains(`Value: ${initialValue}`).should("exist");

      cy.contains("button", "+").click();
      cy.get("div").contains(`Value: ${initialValue + 1}`).should("exist");
  
      cy.contains("button", "+").click();
      cy.get("div").contains(`Value: ${initialValue + 2}`).should("exist");
  
      cy.contains("button", "-").click();
      cy.get("div").contains(`Value: ${initialValue + 1}`).should("exist");
  
      cy.contains("button", "-").click();
      cy.get("div").contains(`Value: ${initialValue}`).should("exist");
  
      cy.get("body").click(0, 0);
    });
  });
});